### ISSUE TITLE
How to import this tool in Eclipse?

### ISSUE THREAD

**Issue by Kaka727** [yyxhdy/arja#3]
Hi @yyxhdy 
I am trying to modify the source code of this tool to do some experiments.
But I find it cannot be imported as Existing projects in Eclipse since "No projects are found to import".
I'd like to know how to modify the code and build a .jar file. Could you please help me?

**Comment by yyxhdy** [comment]
Hi, you can just create an empty Java Project in Eclipse and then put the source code of Arja ("src" folder), dependencies  ("lib" folder), and the external binaries ("external" folder) under this new project. 
And make sure that all jars in the "lib" folder have been linked to your project.

**Comment by Kaka727** [comment]
@yyxhdy Thanks. This method really works.
However, after importing the .jars files, there are still some errors in the code
![image](https://user-images.githubusercontent.com/33283686/59730407-d3df5000-9274-11e9-80a8-790bf43ea0c4.png)

One is about the builder.redirectOutput
![image](https://user-images.githubusercontent.com/33283686/59730437-f1acb500-9274-11e9-8a69-4b9a6075b90e.png)

The other is about the classLoader.close
![image](https://user-images.githubusercontent.com/33283686/59730471-143ece00-9275-11e9-8a0d-53633d404fee.png)

I'd like to know how to solve these problems.

**Comment by Kaka727** [comment]
I find in the external packages, these functions are not defined. 
So is there any difference between the latest code and the previous versions.

**Comment by yyxhdy** [comment]
Can you let me know which JDK version you use in your eclipse project?

**Comment by Kaka727** [comment]
Mine is java 1.8

**Comment by yyxhdy** [comment]
Can you see a jar named "rt.jar" under "JRE System Library"? And make sure that you can 
find the classes "java.net.URLClassLoader" and "java.lang.ProcessBuilder" in rt.jar.

**Comment by Kaka727** [comment]
Yes, I can find these external packages. But I have checked that java.lang.ProcessBuilder does not contain a function named redirectOutput. 
![image](https://user-images.githubusercontent.com/33283686/59732947-ba430600-927e-11e9-82ee-235464bb2ad7.png)

**Comment by yyxhdy** [comment]
The problem may be due to the JDK version. I suggest you use the JDK 1.7 since Arja was not well tested on JDK 1.8.

**Comment by Kaka727** [comment]
Thanks.
After changing the jdk to version 1.7, there is no error now!

**Comment by yyxhdy** [comment]
You are welcome!
